gen3/bin/kube-setup-secrets.sh

```diff
@@ -206,36 +206,48 @@ if [[ -f "$(gen3_secrets_folder)/creds.json" ]]; then # update fence secrets
   if ! g3kubectl get secrets/fence-config > /dev/null 2>&1; then
     # load updated fence-config.yaml into secret if it exists
     fence_config=$(gen3_secrets_folder)/apis_configs/fence-config.yaml
-    if [[ -f ${fence_config} ]]; then
-      echo "loading fence config from file..."
-      if g3kubectl get secrets/fence-config > /dev/null 2>&1; then
-        g3kubectl delete secret fence-config
-      fi
+    fence_pub_config=$(gen3_secrets_folder)/apis_configs/fence-config-public.yaml
+    fence_sec_config=$(gen3_secrets_folder)/apis_configs/fence-config-secret.json
+    if [[ -f ${fence_pub_config} && -f ${fence_sec_config} ]]; then
+      # merge public and secret configs and overwrite ${fence_config}
+      gen3 fence-config merge "${fence_pub_config}"  "${fence_sec_config}" "${fence_config}"
       g3kubectl create secret generic fence-config "--from-file=fence-config.yaml=${fence_config}"
```

Comment:
The merged fence_config should be a temporary file that gets deleted after the k8s secrets is created.